docs/UserGuide.md

```diff
@@ -983,7 +939,7 @@ If your changes to the data file makes its **format invalid**, Foster Family wil
 
 ## **Known issues**
 
-1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The solution is to delete the `preferences.json` file created by the application before running the application again.
+- **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The solution is to delete the `preferences.json` file created by the application before running the application again.
```

Comment:
actually i nvr understood what does off-screen mean HAHAH but i think this is from the original ab3?

Reply:
yeaa it was from the original one. i think it may mean like a part of the window may be cut off once u switch back to using ur main screen only. ill make that part clearer

Reply:
thanks!